app/controllers/conference_controller.rb

```diff
@@ -5,7 +5,7 @@ class ConferenceController < ApplicationController
   load_resource :program, through: :conference, singleton: true, except: :index
 
   def index
-    @current = Conference.where('end_date >= ?', Date.current).order('start_date ASC')
+    @current = Conference.where('end_date >= ?', Date.current).reorder('start_date ASC')
```

Comment:
Turns out it might not be such a good idea to change that in the model. 
Because that would affect other parts of the application, eg. dropdwon with list of conferences in admin panel, and there we don't want to see the oldest conference first. It makes sense seeing the most recent one.

Here in the controller we are only selecting the most recent conferences, and then it makes sense to reorder these as start_date: :asc.

Either way we need to touch something.